lib/screens/settings/dismissible_delete_background.dart

```diff
@@ -7,10 +7,10 @@ class DismissibleDeleteBackground extends StatelessWidget {
   Widget build(BuildContext context) {
     return Container(
         color: Colors.red,
-        child: Padding(
-            padding: const EdgeInsets.symmetric(horizontal: 16),
+        child: const Padding(
```

Comment:
I don't think it's possible to further simplify these const statements. You can't use the const keyword on `Padding` because its child, `Row`, doesn't have a const constructor.

Reply:
I think you are right, I get it wrong sometimes so I will close this pr.